package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "action-cli-deploy",
+  "version": "1.0.0",
+  "description": "GitHub Action to deploy an app using Office 365 CLI",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "ncc build src\\main.ts"
```

Comment:
Do we need tsconfig file if we use ncc to build it?

Reply:
I have left that there in case we decide to have separate tasks for `build` and `pack` like [it is here](https://github.com/actions/typescript-action/blob/master/package.json#L8) in the create typescript action example. If it is not needed then we can remove it.

Reply:
Looks like the `ncc` compiler is using the tsconfig under the hood so we need it.

Reply:
Ah ok. Good to know that.